lib/WP_Auth0_Options_Generic.php

```diff
@@ -22,12 +22,26 @@ public function get_options() {
 		return $this->_opt;
 	}
 
+	/**
+	 * Return a settings value
+	 *
+	 * @param string $key - settings key
+	 * @param null $default - what to return if a value is not set
+	 *
+	 * @return mixed
+	 */
 	public function get( $key, $default = null ) {
 		$options = $this->get_options();
+		$value = $default;
 
-		if ( !isset( $options[$key] ) )
-			return apply_filters( 'wp_auth0_get_option', $default, $key );
-		return apply_filters( 'wp_auth0_get_option', $options[$key], $key );
+		if ( 'passwordless_cdn_url' === $key ) {
```

Comment:
Hence the TODO on the line below. There is another PR for changing how the scripts are being loaded and if I changed it everywhere, I'd have conflicts. I'll clear all the TODOs before release.

Reply:
This is just a stop-gap until #419 is merged. I'll resubmit this once that's been rebased against this one. 

TODOs are just tracked in my IDE. I consider TODOs are requiring resolution before the release. If it's going to go past then, I'll create an issue here or Jira.

Reply:
Fixed this. Instead of deprecating the setting, I'm just replacing it and leaving it in the options array but without a field to update it. I'll merge these two settings once Lock 10 is deprecated and we can assume most folks are on a version that supports passwordless.